### PR TITLE
Prefix template deps, support rewriting dev URLs

### PR DESCRIPTION
**Pull request by mstriemer**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -294,12 +293,16 @@ gulp.task('index_html_build', function() {
         .pipe(gulp.dest('src'));
 });
 
+var rewriteModule = require('http-rewrite-middleware');
```
Comment (ngokevin) [inline index.js]: let's put requires on top

File: index.js [hunk 2]
```diff
@@ -294,12 +293,16 @@ gulp.task('index_html_build', function() {
...
+var rewriteMiddleware = rewriteModule.getMiddleware(config.rewriteMiddleware);
```
Comment (ngokevin) [inline index.js]: no need for intermediate var, we can just throw this right into the webserver opts right?

**Comment by ngokevin** [comment]
looks good

**Comment by mstriemer** [comment]
Merged this into the `tracking-3.0.0` branch. https://github.com/mozilla/marketplace-gulp/commit/434e21aa3f91bec0e19c270646e6cfd587aff757

**Pull request closed without merging**